content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -1,76 +1,86 @@
+See [compatibility with CNIs](#compatibility-with-other-cni-plugins) for details.
+{{< /tip >}}
+
+Follow this guide to install, configure, and use the Istio CNI node agent with the sidecar data plane mode.
+
+## Sidecar traffic redirection without the Istio CNI node agent
 
 By default Istio injects an init container, `istio-init`, in pods deployed in
 the mesh. The `istio-init` container sets up the pod network traffic
 redirection to/from the Istio sidecar proxy. This requires the user or
 service-account deploying pods to the mesh to have sufficient Kubernetes RBAC
 permissions to deploy [containers with the `NET_ADMIN` and `NET_RAW` capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container).
+
+## Sidecar traffic redirection with the Istio CNI node agent
+
 Requiring Istio users to have elevated Kubernetes RBAC permissions is
-problematic for some organizations' security compliance. The Istio CNI plugin
-is a replacement for the `istio-init` container that performs the same
-networking functionality but without requiring Istio users to enable elevated
-Kubernetes RBAC permissions.
+problematic for some organizations' security compliance, as is the requirement to deploy privileged init containers with every workload.
 
-The Istio CNI plugin identifies user application pods with sidecars requiring traffic redirection and
-sets this up in the Kubernetes pod lifecycle's network
-setup phase, thereby removing the [requirement for the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/application-requirements/)
-for users deploying pods into the Istio mesh. The Istio CNI plugin
-replaces the functionality provided by the `istio-init` container.
+The `istio-cni` node agent is effectively a replacement for the `istio-init` container that enables the same
+networking functionality, but without requiring the use or deployment of privileged init containers in every workload.
 
-{{< tip >}}
-Note: The Istio CNI plugin operates as a chained CNI plugin, and it is designed to be used with another CNI plugin,
-such as [PTP](https://www.cni.dev/plugins/current/main/ptp/) or [Calico](https://docs.projectcalico.org).
-See [compatibility with other CNI plugins](#compatibility-with-other-cni-plugins) for details.
-{{< /tip >}}
+Instead, `istio-cni` itself runs as a single privileged pod on the node. It uses this privilege to install a [chained CNI plugin](https://www.cni.dev/docs/spec/#section-2-execution-protocol) on the node, which is invoked after your "primary" interface CNI plugin. CNI plugins are invoked dynamically by Kubernetes as a privileged process on the host node whenever a new pod is created, and are able to configure pod networking.
```

Comment:
Make this paragraph a run-on from the previous (as starting a new section with Instead doesn't read well.)

(GitHub wouldn't let me "suggest" across the blank line that remains, I'm sorry.)